README.md

```diff
@@ -14,3 +14,9 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+The worst-case runtime complexity is $\Theta(n^2)$.  This is because of the two for loops in the mergesort function and the two for loops in the merge function.  I included comments in my code for each line's runtime.
```

Comment:
Not quite -- it might be easier to think about the difference to normal mergesort, in particular where you're doing more work.